src/index.ts

```diff
@@ -1,4 +1,5 @@
 import LocalStorage from './LocalStorage'
+import MemoryStorage from './MemoryStorage'
 
-
+export const memoryStorage = new MemoryStorage()
```

Comment:
I suggest to export MemoryStorage to have ability to control instances creation.